install/kubernetes/cilium/values.yaml

```diff
@@ -401,30 +401,30 @@ bpf:
   preallocateMaps: false
   # -- (int) Configure the maximum number of entries in auth map.
   # @default -- `524288`
```

Comment:
Did you investigate why previously those were set to `nil` with the default value manually documented?

Since it would have been way simpler to just set it to a default value, I do wonder why we previously opted to set it to nil.

Reply:
I guess they we're left at nil because the defaults are being enforced by cilium itself so my guess is that devs wanted to show that is possible to configure it. Another way would've been to put it in the values.yaml and to comment it out.

Reply:
I checked git blame and most were indeed uncommitted to ensure they get picked up by Helm Docs: https://github.com/cilium/cilium/pull/21650

What remains to be discussed is there is any observable change due the value now being present in the configmap (e.g. with regards to downgrades). Let's see what the upgrade/downgrade test in CI says.